### DISCUSSION TITLE
Adding extra information to the Currency class?

### DISCUSSION THREAD

**Discussion by voivoed** [Daniel-Ioannou/flutter_currency_picker#14] [Ideas]
Is it possible to add more information about the currency? I'm looking specifically for decimal places, decimal separator and currency symbol position, so I could format the currency appropriately. The intl package doesn't support a lot of currencies that this one does, I was looking for an alternative to use.

**Comment by Daniel-Ioannou** [comment]
@voivoed  Can you provide a valid source so I can add the info in to Currency?

**Reply by voivoed** [reply]
Hey, I found a couple of sources last night before I wrote but I didn't bookmark anything, and today I can't find them again. Sorry, it's lack of rest due to a big deadline closing in :-/

I did however found this one: https://gist.github.com/Fluidbyte/2973986

Looks pretty nice and has all the info. Not sure if it has all the currencies in the world but the list is way more comprehensive than the one in intl...

**Reply by Daniel-Ioannou** [reply]
Thank @voivoed, I will take a look and with in the week I will add more info in to Currency.
